flink-runtime/src/main/java/org/apache/flink/runtime/io/network/api/serialization/AdaptiveSpanningRecordDeserializer.java

```diff
@@ -181,8 +168,6 @@ public void setReporter(AccumulatorRegistry.Reporter reporter) {
 
 	@Override
 	public void instantiateMetrics(IOMetricGroup metrics) {
```

Comment:
Do we need this method in `RecordDeserializer`? Both implementations of `RecordDeserializer` implement it as empty method.